config/locales/en/placements/schools/placements.yml

```diff
@@ -2,12 +2,25 @@ en:
   placements:
     schools:
       placements:
-        not_entered: Not entered
+        terms:
+          autumn: Autumn
+          spring: Spring
+          summer: Summer
+        not_known_yet: Not known yet
+        window_date: "%{start_month} to %{end_month}"
         index:
           page_title: Placements
           placements: Placements
           subject: Subject
           mentor: Mentor
-          start_date: Start date
+          placement_window: Placement window
```

Comment:
For what it's worth, this has now changed to just "Window" everywhere in the prototype as of https://github.com/DFE-Digital/school-placements-beta-prototype/pull/56.

Reply:
of course it has